config/settings.yml

```diff
@@ -5,6 +5,7 @@ feature_flags:
     production: false
   u18_ioj_passport:
     local: true
+    test: true
```

Comment:
No harm, but not needed, as in test environments all feature flags are enabled by default unless disabled explicitly (i.e. `test: false`)

Reply:
👍 Thank you. I thought it might have a true default,  I didn't know whether that was a known paradigm so decided to explicitly set it. I will revert